fixtures/ed/ntfs_dst/lines.txt

```diff
@@ -1,2 +1,3 @@
 line_id,line_code,line_name,forward_line_name,forward_direction,backward_line_name,backward_direction,line_color,line_text_color,line_sort,network_id,commercial_mode_id,contributor_id,geometry_id,line_opening_time,line_closing_time
-SCF:OCESN8799415287591180,"","Aéroport-CDG-T3 - Gare-de-L'Est-Noctilien","Gare-de-L'Est-Noctilien - Aéroport-CDG-T3",SCF:SA:SAOCE87994152,"Aéroport-CDG-T3 - Gare-de-L'Est-Noctilien",SCF:SA:SAOCE87591180,000000,FFFFFF,1,SNCF,Bus,SCF,,00:02:00,05:08:00
\ No newline at end of file
+SCF:OCESN8799415287591180,"","Aéroport-CDG-T3 - Gare-de-L'Est-Noctilien","Gare-de-L'Est-Noctilien - Aéroport-CDG-T3",SCF:SA:SAOCE87994152,"Aéroport-CDG-T3 - Gare-de-L'Est-Noctilien",SCF:SA:SAOCE87591180,000000,FFFFFF,1,SNCF,Bus,SCF,,00:02:00,05:08:00
+Nav55,"Nav55","Line 55","",,"",,000000,FFFFFF,1,,Bus,,,06:00:00,23:00:00

```

Comment:
minor (not used for tests currently): you may change the line_code so that we test that we use the id properly if we ever test it.
```suggestion
Nav55,"codeNav55","Line 55","",,"",,000000,FFFFFF,1,,Bus,,,06:00:00,23:00:00
```